src/views/dashboard/related/trace/TraceList.vue

```diff
@@ -187,7 +188,7 @@ limitations under the License. -->
   }
 
   .list {
-    width: 280px;
+    width: 100%;
```

Comment:
Please set the `min-width` here.

Reply:
Added, and I modified the default width to 280.